src/ros_to_oscc.cpp

```diff
@@ -64,10 +88,41 @@ void RosToOscc::steeringCommandCallback(const roscco::SteeringCommand::ConstPtr&
   {
     ROS_WARN("OSCC_WARNING occured while trying send the steering torque.");
   }
+
+  // if e_stop then bring the car to a stop.
+  if (e_stop_status_.e_stop) {
```

Comment:
This seems to be duplicated, maybe an internal function like `stop()`? Also, why are we stopping when we retrieve a steering command?

Reply:
The CAN commands are not guaranteed to reach the vehicle (There was congestion from another node publishing too fast, we fixed that, so it might be better now.) . So to give the e-stop command the best chance of being executed as fast as possible, any time a message is being sent on CAN, and e-stop is active, I want to send the e-stop commands too.

Reply:
There is a bit of code duplication with all of the e-stop command sending, I'll probably put that into its own function.